core/src/main/scala/org/bitcoins/core/protocol/tlv/TLV.scala

```diff
@@ -817,6 +818,57 @@ object OracleAnnouncementV0TLV extends TLVFactory[OracleAnnouncementV0TLV] {
   }
 }
 
+sealed trait OracleAttestmentTLV extends TLV {
+  def eventId: NormalizedString
+  def publicKey: SchnorrPublicKey
+  def sigs: Vector[SchnorrDigitalSignature]
+  def outcomes: Vector[NormalizedString]
```

Comment:
super nit (can ignore): But in the case where you make a trait with only one extension such as here where you don't have a future extension in mind, I think it is generally nicer to force people to match to get these `def`s rather than providing them unless you have some reason to believe that future `OracleAttestmentTLV`s are guaranteed to have these fields